var/spack/repos/builtin/packages/py-pyopengl/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import sys
+
+from spack.package import *
+
+
+class PyPyopengl(PythonPackage):
+    """PyOpenGL is the most common cross platform Python binding to OpenGL and related APIs."""
+
+    homepage = "http://pyopengl.sourceforge.net"
+    pypi = "pyopengl/PyOpenGL-3.1.6.tar.gz"
+
+    version("3.1.6", sha256="8ea6c8773927eda7405bffc6f5bb93be81569a7b05c8cac50cd94e969dce5e27")
+
+    variant("glu", default=True, description="Enable OpenGL Utility (GLU) binding.")
+    variant("glut", default=True, description="Enable OpenGL Utility Toolkit (GLUT) binding.")
+
+    conflicts("osmesa", when="^glx")
+    conflicts("glx", when="^osmesa")
```

Comment:
These _should_ be symmetric, so I don't think you actually need both lines.

Reply:
Done